### PR TITLE
feat: Dashboard Responsive Web Develop

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

**Review by jinoosss — COMMENTED** [review]
Nicely done.

I left a few comments,
I think I need to check the tablet-sized data alignment of the table showing the dashboard information.

File: packages/web/src/components/home/token-list-table/TokenListTable.styles.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export const MobileTableHeader = styled.div<{ tdWidth: number }>`
     flex-shrink: 0;
     justify-content: flex-start;
   }
-  &:nth-last-child(1) {
+  &:nth-last-of-type(1) {
     padding: 0px 16px 0px 0px;
   }
   span {
```
[the remark this answers is not in the document]
Comment (juntaepark) [reply packages/web/src/components/home/token-list-table/TokenListTable.styles.ts:104]: Good find.

File: packages/web/src/components/dashboard/activity-info/ActivityInfo.tsx [hunk 2]
```diff
@@ -70,4 +73,60 @@ const ActivityInfo: React.FC<ActivityInfoProps> = ({ item, idx }) => {
...
 
+export const MobileActivityInfo: React.FC<ActivityInfoProps> = ({
+  item,
+  idx,
+}) => {
+  const { action, totalValue, tokenAmountOne, tokenAmountTwo, account, time } =
+    item;
+  const adjective = ["for", "and"];
+  return (
+    <TokenInfoWrapper>
```
[the remark this answers is not in the document]
Comment (juntaepark) [reply packages/web/src/components/dashboard/activity-info/ActivityInfo.tsx:84]: Good find.

File: packages/web/src/components/dashboard/dashboard-info/DashboardInfo.styles.ts [hunk 3]
```diff
@@ -1,13 +1,24 @@
 import styled from "@emotion/styled";
+import { media } from "@styles/media";
+
 import mixins from "@styles/mixins";
 
 export const DashboardInfoWrapper = styled.div`
   ${mixins.flexbox("column", "flex-start", "flex-start")};
-  max-height: 398px;
+  max-width: 1360px;
```
[the remark this answers is not in the document]
Comment (juntaepark) [reply packages/web/src/components/dashboard/dashboard-info/DashboardInfo.styles.ts:8]: Good find.

**Review by jinoosss — APPROVED** [review]
Approve !
It would be nice to manage the content size as a constant variable.

**Pull request merged**